src/core/core.js

```diff
@@ -442,7 +466,8 @@ Crafty.fn = Crafty.prototype = {
             }
         }
         delete this.__c[id];
-
+        // update map from component to (entityId -> entity)
```

Comment:
Oh, I guess it  is technically a lookup of `(entityId -> entity)`, but I don't think that is used anywhere in the code?  (You're entirely concerned with getting the ids, not the entities.)